crates/tabby-scheduler/src/dataset/tags.rs

```diff
@@ -73,6 +73,12 @@ lazy_static! {
                     .unwrap(),
                 ),
             ),
+            (
+                "kotlin",
+                TagsConfigurationSync(
+                    TagsConfiguration::new(tree_sitter_kotlin::language(), "", "").unwrap(),
```

Comment:
Without the tags query, indexing will not function properly (as no tags will be extracted), resulting in the `Repository Context` functionality not working.

For guidance, you can refer to examples such as https://github.com/TabbyML/tabby/blob/main/crates/tabby-scheduler/queries/go.scm and implement it using https://tree-sitter.github.io/tree-sitter/playground.

Alternatively, you can choose to remove this marker and retain only the `Stop Words` section checked in the markdown document.

Reply:
that make sense. I have added kotlin.scm to add the tags query.